### PR TITLE
Inference speedup

### PR DESCRIPTION
**Pull request by teoh**
# PR Ready

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

#### Short description:
BERT similarity model now runs inference faster, thanks to optimizations described [here](https://medium.com/google-cloud/optimizing-tensorflow-models-for-serving-959080e9ddbf). 
Also added a smaller model so that `make test` can run.

### CODE REVIEW

**Comment by teoh** [comment]
tests failed because the model is only accessible on the staging environment (i.e. nlp server), and not in testing or prod environments yet.

File: app/controllers/siamese_bert_similarity.py [hunk 1]
```diff
@@ -34,39 +30,43 @@ def post(self):
...
+    print(f'--- Done loading the model!')
+
+    print('--- computing dummy input... ---')
+    # run in dummy input since model has some overhead on the first prediction
+    dummy_val = predict_fn({
+        'l_input_ids': [[0] * model_configs.MAX_SEQ_LENGTH],
+        'r_input_ids': [[0] * model_configs.MAX_SEQ_LENGTH],
+        'l_input_mask': [[0] * model_configs.MAX_SEQ_LENGTH],
+        'r_input_mask': [[0] * model_configs.MAX_SEQ_LENGTH]})
+    print('--- done computing dummy input... ---')
```
Comment (arunkumar-patange) [inline app/controllers/siamese_bert_similarity.py:49]: instead of print using loggers would be ideal

Comment (teoh) [reply app/controllers/siamese_bert_similarity.py:49]: better to do in this PR or in a separate PR?

Comment (teoh) [reply app/controllers/siamese_bert_similarity.py:49]: talked offline, can do in a separate PR

**Comment by teoh** [comment]
Will add logging in a future PR. Here's the JIRA task: https://brain-ai.atlassian.net/jira/software/projects/FTM/boards/10?selectedIssue=FTM-23

**Comment by asongzz** [comment]
👍

**Pull request merged**